### PR TITLE
Add iris keymap 'boo'

### PR DESCRIPTION
**Pull request by BanchouBoo**
## Description

New keymap for keebio/iris

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by ridingqwerty — APPROVED** [review]

File: keyboards/keebio/iris/keymaps/boo/keymap.c [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                        LT1, LT2, LT3, RT3, RT2, RT1 \
+    ) \
+    LAYOUT( \
+        KC_##L00, KC_##L01, KC_##L02, KC_##L03, KC_##L04, KC_##L05,                     KC_##R00, KC_##R01, KC_##R02, KC_##R03, KC_##R04, KC_##R05, \
+        KC_##L10, KC_##L11, KC_##L12, KC_##L13, KC_##L14, KC_##L15,                     KC_##R10, KC_##R11, KC_##R12, KC_##R13, KC_##R14, KC_##R15, \
+        KC_##L20, KC_##L21, KC_##L22, KC_##L23, KC_##L24, KC_##L25,                     KC_##R20, KC_##R21, KC_##R22, KC_##R23, KC_##R24, KC_##R25, \
+        KC_##L30, KC_##L31, KC_##L32, KC_##L33, KC_##L34, KC_##L35, KC_##LT4, KC_##RT4, KC_##R30, KC_##R31, KC_##R32, KC_##R33, KC_##R34, KC_##R35, \
+                                                KC_##LT1, KC_##LT2, KC_##LT3, KC_##RT3, KC_##RT2, KC_##RT1 \
+    )
+#endif
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/boo/keymap.c:56]:
```suggestion
```

Layouts which prefix keys with KC_ are discouraged now, as they do not play well with the Configurator.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/boo/keymap.c [hunk 2]
```diff
@@ -0,0 +1,245 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/boo/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/boo/keymap.c (hunk 1 above)
Comment (BanchouBoo) [reply keyboards/keebio/iris/keymaps/boo/keymap.c]: Fixed

File: keyboards/keebio/iris/keymaps/boo/keymap.c (hunk 2 above)
Comment (BanchouBoo) [reply keyboards/keebio/iris/keymaps/boo/keymap.c:28]: Added the Unlicense, hope this is acceptable

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**